src/core/qgsactionmanager.h

```diff
@@ -89,6 +89,15 @@ class CORE_EXPORT QgsActionManager: public QObject
     void removeAction( const QUuid &actionId );
 
     /**
+     * Does the action like doAction() providing
+     *  action context scope.
+     */
```

Comment:
Can you add some documentation for parameters? Moreover, `\since QGIS 3.0` is required in this case.